src/components/assets/Assets.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div class="wrapper--assets">
+    <div v-if="!selectedAddress" class="backdrop--transparent" />
+    <div class="container--assets">
+      <Account :ttl-erc20-amount="ttlErc20Amount" />
+      <div v-if="!isLoading && selectedAddress">
```

Comment:
![Screen Shot 2022-03-21 at 9 32 51 AM](https://user-images.githubusercontent.com/40356749/159227540-f25fd530-34d0-48f1-987d-927c9ae1d71d.png)

After logging in, it takes time for the asset panel to show up.
I think it's better to add a loading skeleton for the contents

Reply:
I also want to know my transferrable balance

Reply:
> I also want to know my transferrable balance
@Kahonnohak :)

Reply:
Yes, make sense. Total balance + transferrable or just transferrable might be fine. will get back to you this evening with other feedback thanks.

Reply:
<img width="1014" alt="スクリーンショット 2022-03-22 9 44 26" src="https://user-images.githubusercontent.com/77480847/159385587-d1dc7c66-7513-438b-9b98-c2b44f930e57.png">
@impelcrypto @hoonsubin 
The balance at the top is total balance and the faucet would be here when the balance is zero.
The rest of the lines of the related balance up below. I think it is ok to show zero balance for all (don't have to hide) as then people know that you haven't got any otherwise they would ask. But also don't have to make a change if the current setting is to hide for zero balance.

Reply:
@Kahonnohak Pushed, please take a look at the [staging](https://deploy-preview-pr-282.herokuapp.com/) url.